md/src/main/java/io/codelex/basket/Basket.java

```diff
@@ -0,0 +1,37 @@
+package io.codelex.basket;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Supplier;
+
+public class Basket <T> {
+    List<T> basket = new ArrayList<>();
+
+    private final T item;
```

Comment:
Šis mainīgais būtu jādzēš ārā - jo viņam īsti nav nozīmes, lietas, kuras ir grozā jau glabāsies basket sarakstā.